app/controllers/posts_controller.rb

```diff
@@ -1,7 +1,7 @@
 class PostsController < ApplicationController
   def show
-    @post = Post.where(author_id: params[:user_id], id: params[:id]).first
-    @comments = @post.comments
+    @post = Post.includes(comments: :author).find_by(author_id: params[:user_id], id: params[:id])
+    @comments = @post.comments.includes(:author)
```

Comment:
- [ ] @KIsmoilov and @ndikumanaisaie great work fixing the N+1 problem however you didn't include a screenshot of before and after the fix, the current screenshot are for posts index and show page , kindly include screenshot of before and after fixing the N+ 1 problem